specification/search/data-plane/Azure.Search/preview/2021-04-30-Preview/searchindex.json

```diff
@@ -1529,6 +1772,36 @@
       },
       "description": "This parameter is only valid if the query type is 'semantic'. If set, the query returns captions extracted from key passages in the highest ranked documents. When Captions is set to 'extractive', highlighting is enabled by default, and can be configured by appending the pipe character '|' followed by the 'highlight-<true/false>' option, such as 'extractive|highlight-true'. Defaults to 'None'."
     },
+    "Debug": {
+      "type": "string",
+      "enum": [
+        "disabled",
+        "semantic",
+        "all"
+      ],
+      "x-ms-enum": {
+        "name": "QueryDebugMode",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "disabled",
+            "name": "Disabled",
+            "description": "Do not return debug info for the query."
+          },
+          {
+            "value": "semantic",
+            "name": "Semantic",
+            "description": "Return semantic debug info for the query."
+          },
+          {
+            "value": "all",
+            "name": "All",
+            "description": "Return all debug info for the query."
+          }
+        ]
+      },
+      "description": "This parameter controls debug response information for queries and allows the user to pass in a combination of flags like so: 'Semantic | All'. The values 'semantic' or 'all' are only valid when the query type is 'semantic'."
```

Comment:
Is this a repeat of line 111?   If you took the suggested edit for line 111, can you repeat it here?